tutorials/coral-talk-on-cloud-run/index.md

```diff
@@ -0,0 +1,164 @@
+---
+title: Coral Talk on Google Cloud Run
+description: How to deploy Coral Talk on Google Cloud Platform using managed services -  Cloud Run and Memorystore.
+author: vyolla
+tags: cloud-run, memorystore
+date_published: 2022-04-04
+---
+
+Bruno Patrocinio | Customer Engineer | Google
+
+
+<p style="background-color:#CAFACA;"><i>Contributed by Google employees.</i></p>
+
+This tutorial describes how to deploy open-source commeting platform, [Coral Talk](https://docs.coralproject.net/)  on Google Cloud Platform using managed services.
+
+The diagram below shows the general flow: 
+![architecture](images/architecture.png)
+
+The instructions are provided for a Linux development environment, such as [Cloud Shell](https://cloud.google.com/shell/).
+However, you can also run the application on Google Compute Engine, Kubernetes, a serverless environment, or outside of
+Google Cloud.
+
+This tutorial assumes that you know the basics of the following products and services:
+
+  - [Cloud Run](https://cloud.google.com/run/docs)
+  - [Container Registry](https://cloud.google.com/container-registry/docs)
+  - [Memorystore](https://cloud.google.com/memorystore/docs)
+  - [Compute Engine](https://cloud.google.com/compute/docs)
+  - [`gcloud`](https://cloud.google.com/sdk/docs)
+  - [Docker](https://docs.docker.com/engine/reference/commandline/run)
+
+## Objectives
+
+* Learn how to create and deploy services using `gcloud` commands.
```

Comment:
is this the specific objective for this tutorial? it feels like it's showing an example of an application deployment with Cloud Run and Memory Store?